draft-packages/tile/KaizenDraft/Tile/components/GenericTile.scss

```diff
@@ -15,6 +14,11 @@ $tilePaddingTop: $kz-var-spacing-xl;
 
 .tile {
   width: $tileWidth;
+
+  [class*="TileGrid__grid"] & {
```

Comment:
I do worry about this breaking in the future. Could you use a _known_ attribute on the grid element instead so that the connection is unbreakable?

```
[data-tile-grid] & {
  width: auth
}
```

Reply:
That's a great idea 💡  I was also hoping there was a less brittle way to do this but I didn't think of that! Will update.